src/components/HumanInterface/HumanFront.vue

```diff
@@ -289,6 +294,44 @@ export default defineComponent({
   setup() {
     return { toggleList, chooseMuscle };
   },
+  computed: {
```

Comment:
same thing here, see other comment in `HumanBack.vue`

Reply:
done